e2e/test.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs')
+const path = require('path')
+
+const shell = require('shelljs')
+
+const chimiBin = path.resolve(__dirname, '..', 'bin', 'bin.js')
+const fixturesRoot = path.join(__dirname, 'fixtures')
+
+const fixturesDirs = fs
+  .readdirSync(fixturesRoot)
+  .map(x => path.join(fixturesRoot, x))
+  .filter(x => fs.lstatSync(x).isDirectory())
+
+describe('e2e tests', () => {
+  for (let fixtureDir of fixturesDirs) {
+    const relativeFixtureDir = path.relative(fixturesRoot, fixtureDir)
+    it(`should run ${relativeFixtureDir} as expected`, () => {
+      shell.cd(fixtureDir)
+
+      const execution = shell.exec(`node ${chimiBin}`, { silent: true })
+      const result = [execution.code, execution.stdout]
```

Comment:
If we do that, then where does the expectation "lives"? Unless we separate the tests in "pass" (return 0) and "fail" (return 1), or put the expected status in the filename (over my dead body).

Reply:
I'd do the first, for sure.

Another option is to have a JSON file in each of the case's folders with the expectation value. We could put more information there, like the case's name (instead of getting it from the directory name).

Reply:
Let's do the JSON approach.